### DISCUSSION TITLE
Mod Support?

### DISCUSSION THREAD

**Discussion by Darkerowl** [gri573/rethinking-voxels#265] [General]
Hello, 
On version R0.1-Alpha5 in changelog for Modrinth you say • Remove all mod Support, was that ever reinstated to allow this amazing shader to be used with mods or modpacks Like Better MC BMC4 is one i'm on. I used this back on alpha 4 with the issue in the nether but now that pipeline is worked I want to use this shader again?!

**Comment by gri573** [comment]
mod support will be back in the next version
